Remotion/Web/Development.WebTesting/BrowserSession/BrowserSessionBase.cs

```diff
@@ -74,7 +74,7 @@ public void AcceptModalDialog (Options options = null)
       _value.AcceptModalDialog (options);
     }
 
-    public Driver Driver
+    public IDriver Driver

```

Comment:
doc breaking change

Reply:
Added [RM-7282](https://re-motion.atlassian.net/browse/RM-7282).
Waiting for the the completion of this PR before adding the fix version & closing it.